src/runner.rs

```diff
@@ -238,13 +238,26 @@ impl Interpreter {
     }
 
     fn get_tracer_if_active(&self) -> Option<Rc<RefCell<Tracer>>> {
```

Comment:
name of the function does not imply the logic clearly.  the logic is get tracer if neither masked or in counting phase?